motia-content-creation/steps/generate-twitter.step.py

```diff
@@ -3,13 +3,11 @@
 from pydantic import BaseModel, HttpUrl
 from datetime import datetime
 from dotenv import load_dotenv
-from openai import AsyncOpenAI
+import ollama
 
 load_dotenv()
 
-OPENAI_API_KEY = os.getenv('OPENAI_API_KEY')
-
-openai_client = AsyncOpenAI(api_key=OPENAI_API_KEY)
+OLLAMA_MODEL = os.getenv('OLLAMA_MODEL', 'deepseek-r1')
```

Comment:
Change the default ollama model to Qwen3 instad of Deepseek-R1
```suggestion
OLLAMA_MODEL = os.getenv('OLLAMA_MODEL', 'qwen3:4b')
```